js/indicators/awesome.src.js

```diff
@@ -0,0 +1,197 @@
+'use strict';
+import H from '../parts/Globals.js';
+import '../parts/Utilities.js';
+
+var correctFloat = H.correctFloat,
+    isArray = H.isArray,
+    noop = H.noop;
+
+H.seriesType('awesome', 'sma',
```

Comment:
Awesome is too general. Consider to use different name of series, i.e "ao" its shortcut like "sma", "ema" etc.

Reply:
Also change name of file.